### DISCUSSION TITLE
Can I join your Organization?

### DISCUSSION THREAD

**Discussion by zoraezimran** [Bijoujs/Bijou.js#157] [Q&A] [answered]
I would like to join this company because I have a pack of lots of good code snippets.

Pls chose me for this

Pls let me join this organization

**Comment by ghost** [comment]
@Explosion-Scratch  this call's for you

**Reply by zoraezimran** [reply]
pls can you get @Explosion-Scratch fast

**Reply by retronbv** [reply]
> pls can you get @Explosion-Scratch fast

Its 6 in the morning for them, can you share some of your snippets while we wait? (Do you have discord?)

**Reply by zoraezimran** [reply]
I don't have discord

**Reply by zoraezimran** [reply]
@retronbv can you get @Explosion-Scratch as soon as you know he is avaible so I can join your organization or since you are the maintainer as well you  can get me in the organization

**Reply by retronbv** [reply]
I will ask him

**Reply by zoraezimran** [reply]
ok thanks

**Reply by retronbv** [reply]
![DF5A3D4E-445C-4F55-AB7F-5805F23CCA56](https://user-images.githubusercontent.com/49005044/114720353-f40fb700-9d05-11eb-9b6e-cb9bbb923861.jpeg)
no response yet

**Reply by zoraezimran** [reply]
ok

**Reply by Explosion-Scratch** [reply]
Sorry guys! Adding you! 😄

**Reply by zoraezimran** [reply]
no prob

**Comment by Explosion-Scratch** [comment, marked as answer]
Added

**Reply by zoraezimran** [reply]
thanks

**Comment by Explosion-Scratch** [comment]
@ZI-Youtuber

**Comment by xItzDiamondx** [comment]
Hey @Explosion-Scratch, can you please add my test @NotDiamondZ because I have too many repos on here so I'm gonna fork it on that acc. Thanks.

**Reply by Explosion-Scratch** [reply]
Sure thing!

**Comment by xItzDiamondx** [comment]
Thanks! :D @NotDiamondZ is my test and I already have to many repos on this
acc. So thats why I'm asking.

**Reply by Explosion-Scratch** [reply]
Np!
